.travis.yml

```diff
@@ -19,11 +28,11 @@ install:
 before_script:
   - export PYTHONPATH=$PYTHONPATH:$(pwd)
   - export PYRO_HOME=$(pwd)
-  - ./mk.sh
+  - ./mk.sh $MK_ARGS
 
 script:
-  - flake8 .
-  - pytest -v --cov=. --cov-config .coveragerc --nbval --ignore=docs --ignore=./multigrid/variable_coeff_elliptic.ipynb --ignore=examples/mesh --ignore=examples/multigrid
+  - flake8 --ignore=W504,F403,E226,E126,E127,E128,E129,E241,E741 .
```

Comment:
there already is a `.flake8`, so maybe we can add the ignores there?

Reply:
Fixed by 69dec2946